### PR TITLE
added config path override for custom openzwave config files / modifications.

### PR DESCRIPTION
**Pull request by LennardWesterveld**
# Proposed Changes

> (Describe the changes and rationale behind them)

## Related Issues

> ([Github link][autolink-references] to related issues or pull requests)

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

**Comment by LennardWesterveld** [comment]
Still not sure if this would work, i'm also searching for a solution to change the config file of openzwave schould be the best solution i guess? so just change the location where openzwave looks op the config files.

**Comment by frenck** [comment]
Please clarify: "Still not sure if this would work" ?

**Comment by LennardWesterveld** [comment]
Tested and it works on my local hassio setup, only if the docker builds get fixed of course but i know that Frenck is working on it.

**Comment by sebastian-steinmann** [comment]
Seems like z2m v3 will have this as a built in option. Override the devices.js and json-format. But I might be misunderstanding.

**Comment by Munsio** [comment]
From the readme of zwave2mqtt repository:

> UPDATE: Starting from version 2.0.7 you can specify your custom devices configuration inside `store/customDevices(.js|.json)` file. This allows users that use Docker to create their custom hass devices configuration without the need to build a new container. If using .json format Zwave2Mqtt will watch for file changes and automatically load new components on runtime without need to restart the application.

So i think this PR can be closed.

File: README.md [hunk 1]
```diff
@@ -118,6 +118,14 @@ username and password empty.
 **Note**: _We STRONGLY suggest, not to use this, even if this add-on is
 only exposed to your internal network. USE AT YOUR OWN RISK!_
 
+### Option: `config_path`
+
+Adding this option to override the OpenZWave config directory. To add for example custom configs for not supported devices.
+
+**Note**: You need to download the full OpenZWave db from https://github.com/OpenZWave/open-zwave/tree/1.4 and place the `config` content a persistent location like the `/config
```
Comment (frenck) [inline README.md:125]: We are running 1.6.

File: zwave2mqtt/rootfs/etc/cont-init.d/zwave2mqtt.sh [hunk 2]
```diff
@@ -7,3 +7,8 @@
 # Ensure store is persistent
 rm -f -r /opt/store
 ln -s /data /opt/store
+
+if bashio::config.exists 'config_path'; then
+    rm -f -r /usr/etc/openzwave
```
Comment (frenck) [inline zwave2mqtt/rootfs/etc/cont-init.d/zwave2mqtt.sh:12]: This path might not be right, which we are looking into right now. See #36

**Comment by frenck** [comment]
Yeah it has been build-in now, so we can close this PR.

**Pull request closed without merging**